lib/packwerk/files_for_processing.rb

```diff
@@ -47,19 +49,21 @@ def files
 
     private
 
-    sig { returns(T::Array[String]) }
+    sig { returns(AbsoluteFileSet) }
     def custom_files
-      @custom_files ||= @relative_file_paths.flat_map do |relative_file_path|
-        absolute_file_path = File.expand_path(relative_file_path, @configuration.root_path)
-        if File.file?(absolute_file_path)
-          absolute_file_path
-        else
-          custom_included_files(absolute_file_path)
+      @custom_files ||= Set.new(
+        @relative_file_paths.flat_map do |relative_file_path|
```

Comment:
Is this `flat_map` still relevant? Since the return of this block is sometimes a set isn't doing anything for that case anymore, hence the need to call `flatten` after the set is built. Maybe we should only call `flatten` after the set is built?

Reply:
Agreed! Changed.